server.js

```diff
@@ -31,12 +100,17 @@ app.post("/user/reset", function(req, res) {
   res.json({ info: "user list cleared" });
 });
 
-let gTransporter;
+// This exists only right now for development purposes.
```

Comment:
nit: Not sure if we want to add a "// TODO" or some follow up bug to remove this route in the future. Having this endpoint sneak onto production makes me a bit uneasy.

Reply:
This will not sneak into production. As soon as we add persistent storage we won't need this, and removing this API will be part of the process of removing the gEmails Set.